src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -264,7 +264,7 @@ public override BoundNode VisitLambda(BoundLambda node)
 
         public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatement node)
         {
-            _sawLocalFunctions = true;
+            int localFunctionOrdinal = _availablelocalFunctionOrdinal++;
```

Comment:
This doesn't need to be atomic, right?

Reply:
> For my own clarifiction, this doesn't need to be atomic, right?

It is atomic, I believe.

---
In reply to: [537700973](https://github.com/dotnet/roslyn/pull/49802#discussion_r537700973) [](ancestors = 537700973)